src/main/kotlin/it/pagopa/helpdeskcommands/services/CommandsService.kt

```diff
@@ -51,7 +51,7 @@ class CommandsService(
                         grandTotal = amount,
                         apikey = apiKey,
                         description =
-                            "Refund request for transactionId $transactionId and operationId $operationId"
+                            "Refund request for transactionId ${transactionId.uuid} and operationId $operationId"
```

Comment:
```suggestion
                            "Refund request for transactionId [${transactionId.value}] and operationId [$operationId]"
```

Reply:
this is a description field for the refund operation, just use the eCommerce transactionId format here just to be coherent with the eCommerce saved one in case of issues with NPG

Reply:
done!
thanks for the hint